tests/integration/testlib/cmd.py

```diff
@@ -41,7 +41,7 @@ def exec_cmd(cmd, env=None):
 
     logging.debug(_retcode_log_line(p.returncode, err=err))
 
-    return p.returncode, out, err
+    return p.returncode, out.decode('utf-8'), err.decode('utf-8')
```

Comment:
In production code I would have asked to create a wrapper to convert bytes to unicode, but for tests it should be fine.
(reasoning that the caller may actually want to process bytes)

Anyway, it will be nice if you add a word or two in the doctext to mention that the output is unicode.